index.js

```diff
@@ -0,0 +1,139 @@
+
+// ## Deliverable 1: Fetch Players
+
+    // access the player container div using querySelector() method and set it to a variable
+const playerContainer = document.querySelector(".player-container")
+
+    // render one player to the DOM
+function renderPlayer(player) {
+        // create an element for the outer div
+    const playerDiv = document.createElement("div")
+
+        // set attributes on the outer div
+    playerDiv.className = "player"
+    playerDiv.dataset.id = player.id
```

Comment:
Nice job setting up this `data-id` attribute so you can access it with the event delegation 👍